content/02.main-text.md

```diff
@@ -168,11 +169,12 @@ The idea of the "priority of discovery" is important to science, and Vale and Hy
 In their framework, disclosure occurs when a scientific output is released to the world.
 However, for a manuscript that is shared as it is written, being able to establish priority could be challenging.
 Manubot supports [OpenTimestamps](https://opentimestamps.org/) to timestamp the HTML and PDF outputs on the Bitcoin blockchain.
-This procedure allows one to retrospectively prove that a manuscript version existed prior to its blockchain-verifiable timestamp [@arxiv:1502.04015v1; @url:https://www.bgcarlisle.com/blog/2014/08/25/proof-of-prespecified-endpoints-in-medical-research-with-the-bitcoin-blockchain/; @url:http://blog.dhimmel.com/irreproducible-timestamps/; @url:http://git.dhimmel.com/bitcoin-whitepaper/].
+This procedure allows one to retrospectively prove that a manuscript version existed prior to its blockchain-verifiable timestamp [@arxiv:1502.04015v1; @url:https://www.bgcarlisle.com/blog/2014/08/25/proof-of-prespecified-endpoints-in-medical-research-with-the-bitcoin-blockchain/; @url:http://blog.dhimmel.com/irreproducible-timestamps/; @doi:10.1038/d41586-019-00447-9; @url:http://git.dhimmel.com/bitcoin-whitepaper/].
 Timestamps protect against attempts to rewrite a manuscript's history and ensure accurate histories, potentially alleviating certain authorship or priority disputes.
 Because all bitcoin transactions compete for limited space on the blockchain, the fees required to send a single transaction can be high.
-OpenTimestamps avoids this fee by encoding many timestamps into a single Bitcoin transaction [@url:https://petertodd.org/2016/opentimestamps-announcement].
-There can be a lag of a few hours before the transaction is made, which is suitable for the purposes of scientific writing.
+OpenTimestamps minimizes fees by encoding many timestamps into a single Bitcoin transaction, enabling the service to be free of charge [@url:https://petertodd.org/2016/opentimestamps-announcement].
+Since transactions can take up to a few days to be made, Manubot initially stores incomplete timestamps and upgrades them in future continuous deployment builds.
```

Comment:
I prefer "Because" to "Since" in this context.

This sentence is jargon-heavy but difficult to simplify.

Reply:
Note that two sentences before begins with "Because all bitcoin transactions compete"

Reply:
Okay, this appears to be my own personal writing quirk rather than any generally accepted grammatical rule.  We can leave it.